invertedai/api/drive.py

```diff
@@ -102,15 +102,19 @@ def drive(
 
     traffic_lights_states:
        If the location contains traffic lights within the supported area,
-       their current state should be provided here. Any traffic light for which no
-       state is provided will have a state generated by iai.
+       their current state can be provided here. It is legal to not provide this field, and use
+       'light_recurrent_states' to step the traffic lights. If provided, light states from 'traffic_light_states' will override
+       the original light states given by 'light_recurrent_states'. The server does not currently support continuing user-provided light state sequences, 
+       so once the states are provided at any step, they should also be provided on all subsequent steps to guarantee coherent light sequences.
+       If neither 'traffic_lights_states' nor 'light_recurrent_states' are provided, the server will realistically initialize the traffic light states,
```

Comment:
```suggestion
       If neither 'traffic_lights_states' nor 'light_recurrent_states' are provided, the server will arbitrarily initialize the traffic light states,
```
I don't think the server will consider the current vehicle positions when setting the light states, so I wouldn't call it realistic.